frontend/src/Context/PublisherProvider/usePublisher/usePublisher.tsx

```diff
@@ -242,11 +242,7 @@ const usePublisher = (): PublisherContextType => {
         return;
       }
 
-      mSession.session.publish(publisherRef.current, (error) => {
-        if (error) {
-          throw new Error(`${error.name}: ${error.message}`);
-        }
-      });
+      mSession.publish(publisherRef.current);
```

Comment:
The error handling is in the `VonageVideoClient` class. It can throw, but the error will be caught and handled below.